src/py/flwr/cli/new/new.py

```diff
@@ -37,6 +37,7 @@ class MlFramework(str, Enum):
     NUMPY = "NumPy"
     PYTORCH = "PyTorch"
     TENSORFLOW = "TensorFlow"
+    MLX = "MLX"
```

Comment:
Maybe it's worth listing the available frameworks in alphabetical order? So `MLX` would go before `NUMPY` on line 37.

Reply:
I added sorting when displaying the options, so we can have the frameworks listed in alphabetical order without having to keep the enum sorted manually